### PR TITLE
Cleanup interop code in System.Drawing

### PR DESCRIPTION
**Pull request by hughbe**

### CODE REVIEW

File: src/System.Drawing.Common/src/System/Drawing/Gdiplus.cs [hunk 1]
```diff
@@ -274,301 +252,298 @@ private struct StartupOutput
...
             {
-                if (!Initialized) return SafeNativeMethods.Gdip.Ok;
-                int result = IntGdipDeletePathIter(pathIter);
-                return result;
+                if (!Initialized)
+                {
+                    return Ok;
+                }
+
+                return IntGdipDeletePathIter(pathIter);
```
Comment (stephentoub) [inline src/System.Drawing.Common/src/System/Drawing/Gdiplus.cs]: Nit: the several functions of this form could be condense to, e.g.
```C#
internal static int GdipDeletePathIter(HandleRef pathIter) => Initialized ? IntGdipDeletePathIter(pathIter) : Ok;
```

Comment (hughbe) [inline src/System.Drawing.Common/src/System/Drawing/Gdiplus.cs]: Sure, done

**Comment by mellinoe** [comment]
Thanks for the cleanup @hughbe . I will take a close look at this in a little bit.

File: src/System.Drawing.Common/src/misc/DebugHandleTracker.cs [hunk 2]
```diff
@@ -154,15 +147,15 @@ private class HandleType
             private int _handleCount;
             private HandleEntry[] _buckets;
 
-            private const int BUCKETS = 10;
+            private const int NumberOfBuckers = 10;
```
Comment (mellinoe) [inline src/System.Drawing.Common/src/misc/DebugHandleTracker.cs]: typo: NumberOfBuckers -> NumberOfBuckets.

**Comment by hughbe** [comment]
>  I noticed one type that we should fix, but other than that it's ready to go.

I believe you may have a typo in your typo! Just pushed a fix, sorry this is blocking your other PR

**Comment by mellinoe** [comment]
Doh! Latest version looks good to me 😄

**Pull request merged**